test/browser/integration/caniuse.js

```diff
@@ -17,6 +17,7 @@ window.caniusecb = function(caniuse) {
     // the docs where property and caniuse tags are written down. One should anyway look if some are missing here or in
     // the feature detect docs
     var map = {
+      mediastream: 'mdn-api_mediarecorder',
```

Comment:
Take care, mdn caniuse queries are not supported by Modernizr (due to technical difficulties), but caniuse already has a native mediarecorder table. You actually used it on mediarecorder.js. It is `mediarecorder` (so the two sides of the line should be the same*)